src/libs/TransactionRecoveryApi/processors/index.ts

```diff
@@ -11,13 +11,14 @@ export default class AxelarGMPRecoveryProcessor {
     src: EvmChain;
     dest: EvmChain;
     debug?: boolean;
-  }): Promise<"triggered relay" | "approved but not executed" | "already executed" | unknown> {
+  }): Promise<"triggered relay" | "approved but not executed" | "already executed" | "error_fetching_status"> {
```

Comment:
Convert all of errors into enum? for example,

```ts
export enum RecoveryError {
   TRIGGERED = "triggered relay"; // i'm not sure what this means
   APPROVED = "already approved";
   EXECUTED = "already executed";
   FETCH_ERROR = "unable to fetch status"
}
```


Then, we return with `Promise<RecoveryError>`

Reply:
done